src/notification-banner/private/BannerRegistrationContext.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * The default value for the BannerRegistrationContext includes a flag to indicate that the value is the default value,
+ * which can be used to detect whether any matching context Providers are rendered. Also included are no-op implementations
+ * for register/unregisterBanner to minimize branching logic in the context consumers.
```

Comment:
It doesn't look like that context has been given a default value.

Reply:
Would you expect a default value of `React.createContext({});`?

Reply:
I'd prefer to have an undefined value so that we can detect (and throw an error) when there's no provider in place.